routes/project/user.rb

```diff
@@ -31,6 +31,11 @@ class Clover
           end
 
           tag = dataset_authorize(@project.subject_tags_dataset, "SubjectTag:add").first(name: policy)
+          unless tag || policy
```

Comment:
I'm not sure this condition is correct.  I think the correct condition is to show an error if a policy is selected, but the user does not have access to the subject tag:

```suggestion
          if policy && !tag
```

Reply:
When the user select `No policy`, the `policy` is `""`, so it doesn't falsy. So `tag` is `nil` for `No policy` too.

When the user select a disabled policy from selector, the `policy` is `nil`.

I tried to detect this condition in the old check. But after your comment, I found another issue with that.

So I think the correct version is `if !tag && policy != ""` or `unless tag || policy == ""`